### PR TITLE
ODAA-138: style show flow

### PR DESCRIPTION
**Pull request by yepzdk**
https://jira.itkdev.dk/browse/ODAA-138

### CODE REVIEW

File: templates/data_flow/show.html.twig [hunk 1]
```diff
@@ -6,74 +6,82 @@
...
+    <p class="text-muted small mb-1">{{ 'Flow name'|trans }}</p>
+    <h2 class="h5">{{ data_flow.name }}</h2>
+
+    {% if data_flow.description %}
+        <div class="col-12 bg-white shadow-sm rounded-lg">
+            <p class="text-muted small mb-1">{{ 'Description'|trans }}</p>
+            <p class="lead">{{ data_flow.description }}</p>
+        </div>
+    {% endif %}
+    {% embed '_table.html.twig' with {'hover': 'null'} %}
```
Comment (rimi-itk) [inline templates/data_flow/show.html.twig]: I suppose you mean (the value) `null` as opposed to (the string) `'null`' here. And then you could just use `{% embed '_table.html.twig' %}` and `hover|default('table-hover')` in the embedded template will handle the missing value correctly.

If you're trying to tell the embedded template to *not use any hover* when passing `{hover: null}`, then you need to use something like `{{ hover is defined and hover is null ? '' : hover|default('table-hover') }}`.

Also, you don't have to write `{'hover': …}` – `{hover: …}` will suffice.

File: templates/data_flow/show.html.twig [hunk 2]
```diff
@@ -6,74 +6,82 @@
...
-            <tr>
                 <th>{{ 'Created at'|trans }}</th>
-                <td>{{ data_flow.createdAt ? data_flow.createdAt|date('Y-m-d H:i:s') : '' }}</td>
+                <th>{{ 'Updated at'|trans }}</th>
             </tr>
+        {% endblock %}
+        {% block tbody %}
             <tr>
-                <th>{{ 'Updated at'|trans }}</th>
+                <td>{{ data_flow.lastRunAt ? data_flow.lastRunAt|date('Y-m-d H:i:s') : 'Never' }}</td>
```
Comment (rimi-itk) [inline templates/data_flow/show.html.twig]: Missing `|trans`.

**Review by rimi-itk — CHANGES_REQUESTED** [review]

File: templates/data_flow/show.html.twig (hunk 1 above)
Comment (yepzdk) [reply templates/data_flow/show.html.twig]: Thanks for clarifying. Wasn't sure how to correctly approach this.

File: templates/data_flow/show.html.twig (hunk 2 above)
Comment (yepzdk) [reply templates/data_flow/show.html.twig]: Thanks. Missed that.

**Comment by yepzdk** [comment]
@rimi-itk new commit to deal with your comments.

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**